doc/about.rst

```diff
@@ -96,86 +96,176 @@ following paper:
    :align: center
-   :target: https://cds.nyu.edu/mooresloan/
+   :target: https://www.microsoft.com/
 
+.. figure:: images/bcg.png
+   :width: 100pt
+   :align: center
+   :target: https://www.bcg.com/beyond-consulting/bcg-gamma/default.aspx
 
-`Télécom Paristech <https://www.telecom-paristech.fr/>`_ funded Manoj Kumar (2014),
-Tom Dupré la Tour (2015), Raghav RV (2015-2017), Thierry Guillemot (2016-2017)
-and Albert Thomas (2017) to work on scikit-learn.
+**Gold Members**
 
-.. image:: themes/scikit-learn/static/img/telecom.png
+.. figure:: images/axa.png
+   :width: 50pt
+   :align: center
+   :target: https://www.axa.fr/
+
+.. figure:: images/bnp.png
    :width: 100pt
    :align: center
-   :target: https://www.telecom-paristech.fr/
+   :target: https://www.bnpparibascardif.com/
 
+.. figure:: images/intel.png
+   :width: 70pt
+   :align: center
+   :target: https://www.intel.com/
 
-`Columbia University <https://columbia.edu/>`_ funds Andreas Müller since 2016.
+.. figure:: images/nvidia.png
+   :width: 70pt
+   :align: center
+   :target: https://www.nvidia.com/
 
-.. image:: themes/scikit-learn/static/img/columbia.png
-   :width: 100pt
+**Silver Members**
+
+.. figure:: images/dataiku.png
+   :width: 70pt
+   :align: center
+   :target: https://www.dataiku.com/
+
+........  
+
+.. figure:: themes/scikit-learn/static/img/columbia.png 
+   :width: 50pt
    :align: center
    :target: https://www.columbia.edu/
 
-Andreas Müller also received a grant to improve scikit-learn from the `Alfred P. Sloan Foundation <https://sloan.org>`_ in 2017.
+   `Columbia University <https://columbia.edu/>`_ funds Andreas Müller since 2016
+
+..........
 
-.. image:: images/sloan_banner.png
-   :width: 200pt
+.. figure:: images/sloan_banner.png
+   :width: 100pt
    :align: center
    :target: https://sloan.org/
 
-`The University of Sydney <https://sydney.edu.au/>`_ funds Joel Nothman since July 2017.
+   Andreas Müller received a grant to improve scikit-learn from the
+   `Alfred P. Sloan Foundation <https://sloan.org>`_ .
+   This grant supports the position of Nicolas Hug and Thomas J. Fan.
 
-.. image:: themes/scikit-learn/static/img/sydney-primary.jpeg
-   :width: 200pt
+...........
+
+.. figure:: themes/scikit-learn/static/img/sydney-primary.jpeg
+   :width: 100pt
    :align: center
    :target: https://sydney.edu.au/
 
-`The Labex DigiCosme <https://digicosme.lri.fr>`_ funded Nicolas Goix (2015-2016),
-Tom Dupré la Tour (2015-2016 and 2017-2018), Mathurin Massias (2018-2019) to work part time
-on scikit-learn during their PhDs. It also funded a scikit-learn coding sprint in 2015.
+   `The University of Sydney <https://sydney.edu.au/>`_ funds Joel Nothman since
+   July 2017.
 
-.. image:: themes/scikit-learn/static/img/digicosme.png
-   :width: 200pt
+............
+
+.. figure:: images/anaconda.png
+   :width: 100pt
+   :align: center
+   :target: https://sydney.edu.au/
+
+   `Anaconda, Inc <https://www.anaconda.com/>`_ funds Adrin Jalali since .
```

Comment:
@adrinjalali could you please check? Thanks!

Reply:
since 2019 (July)

Reply:
Thanks!!